app/Http/Controllers/Company/Adminland/AdminWorkFromHomeController.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace App\Http\Controllers\Company\Adminland;
+
+use Inertia\Inertia;
+use Inertia\Response;
+use Illuminate\Http\Request;
+use App\Helpers\InstanceHelper;
+use Illuminate\Http\JsonResponse;
+use App\Helpers\NotificationHelper;
+use App\Http\Controllers\Controller;
+use App\Http\ViewHelpers\Adminland\AdminWorkFromHomeViewHelper;
+use App\Services\Company\Adminland\WorkFromHome\ToggleWorkFromHomeProcess;
+
+class AdminWorkFromHomeController extends Controller
+{
+    /**
+     * Show the work from home page.
+     *
+     * @return Response
+     */
+    public function index(): Response
+    {
+        $company = InstanceHelper::getLoggedCompany();
+
+        $details = AdminWorkFromHomeViewHelper::index($company);
+
+        return Inertia::render('Adminland/WorkFromHome/Index', [
+            'notifications' => NotificationHelper::getNotifications(InstanceHelper::getLoggedEmployee()),
+            'process' => $details,
+        ]);
+    }
+
+    /**
+     * Toggle the work from home setting in the company.
+     *
+     * @param Request $request
+     * @param int $companyId
+     * @return JsonResponse
+     */
+    public function store(Request $request, int $companyId): JsonResponse
```

Comment:
It should be an `update` (`PUT` request). Of course you'll have to change the request in vue file and the web.php...
```suggestion
    public function update(Request $request, int $companyId): JsonResponse
```